Scripts/ObjCCBORAssertAllSymbolsAreMangled.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# This script tries to find symbols that are not prefixed with the custom build
+# setting OBJC_CBOR_MANGLING_PREFIX. See section "Mangling" in README.md for
+# details.
+
+set -Euo pipefail
+
+nm -gUPA "$BUILT_PRODUCTS_DIR/libObjCCBOR.a" | cut -d ' ' -f 2 > "$TARGET_TEMP_DIR/ObjCCBOR-symbols.txt"
+nm -gUPA "$BUILT_PRODUCTS_DIR/libtinycbor.a" | cut -d ' ' -f 2 > "$TARGET_TEMP_DIR/tinycbor-symbols.txt"
+
+grep -vf "$TARGET_TEMP_DIR/tinycbor-symbols.txt" "$TARGET_TEMP_DIR/ObjCCBOR-symbols.txt" \
+    | grep -v -E "___block_descriptor_" \
+    | grep -v -E "___copy_helper_" \
+    | grep -v -E "___destroy_helper_" \
+    > "$TARGET_TEMP_DIR/ObjCCBOR-relevant-symbols.txt" \
+    ;
+
+cat "$TARGET_TEMP_DIR/ObjCCBOR-relevant-symbols.txt" \
+    | sed 's/__OBJC_LABEL_PROTOCOL_\$//' \
+    | sed 's/__OBJC_PROTOCOL_\$//' \
+    | sed 's/__OBJC_PROTOCOL_REFERENCE_\$//' \
+    | sed 's/_OBJC_CLASS_\$//' \
+    | sed 's/_OBJC_METACLASS_\$//' \
+    | sed -E 's/^\_//' \
+    | uniq \
```

Comment:
It doesn't look like it matters, but I'd consider adding a `sort` to be safe.

```suggestion
    | sort \
    | uniq \
```